packages/mobile/src/navigator/Navigator.tsx

```diff
@@ -592,25 +600,56 @@ export function MainStackScreen() {
       pincodeType,
       account,
       hasSeenVerificationNux,
+      shouldSkipOnboardingEducationScreen,
     } = mapStateToProps(store.getState())
 
+    // Remove Onboarding Education Screen Experiment: Because remote configs are fetched after the initial route is launched,
+    // The randomization is hardcoded here to achieve a 50/50 split, the value is written into the redux store so the same experience
+    // would persist. This block of code should be removed when the experiment is done.
+    let _shouldSkipOnboardingEducationScreen = shouldSkipOnboardingEducationScreen
+    if (shouldSkipOnboardingEducationScreen == undefined) {
+      if (Math.random() < 0.5) {
+        _shouldSkipOnboardingEducationScreen = true
+        dispatch(skipOnboardingEducationScreen(true))
+      } else {
+        _shouldSkipOnboardingEducationScreen = false
+        dispatch(skipOnboardingEducationScreen(false))
+      }
+    }
+
     let initialRoute: InitialRouteName
 
     if (!language) {
       initialRoute = Screens.Language
     } else if (!name || !acceptedTerms || pincodeType === PincodeType.Unset) {
       // User didn't go far enough in onboarding, start again from education
-      initialRoute = Screens.OnboardingEducationScreen
-    } else if (!account) {
-      initialRoute = choseToRestoreAccount
-        ? Screens.ImportWallet
+
+      ValoraAnalytics.track(
+        _shouldSkipOnboardingEducationScreen
+          ? OnboardingEvents.onboarding_education_skipped
+          : OnboardingEvents.onboarding_education_not_skipped
```

Comment:
why are these two different events? Wouldn't it be simpler to have a single event with a boolean field on it indicating whether education was skipped?

Reply:
this can be done with CleverTap